### ISSUE TITLE
Invar gears, when broken, drop 'broken gears' which are made of iron, not invar

### ISSUE THREAD

**Issue by wrincewind** [dalapo/FactoryTech#80]
Where's the nickel go? It'd make more sense for it to drop Broken Invar Gears that give back invar nuggets, no?

**Comment by dalapo** [comment]
Yes, I agree that this should be the case. I don't know how smoothly I could implement it, but it is something that I'll address.

**Comment by wrincewind** [comment]
Cool, glad to hear. Thanks for this mod, by the way, I'm really enjoying it!
